app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,21 @@
+module SessionsHelper
+    def log_in(user)
```

Comment:
インデントは基本半角スペース2つに揃えたほうがいいかな 👀 
```suggestion
module SessionsHelper
  def log_in(user)
```

Reply:
このファイルもそうだし、他のファイルもちょこちょこ4つがありそう

Reply:
```
Rails.application.routes.draw do
  root   'static_pages#home'
  
  resources :static_pages do
    collection do
      get :help
      get :about
      get :contact
    end
  end

  resources :users do
    collection do
      get :signup
      post :signup
    end
  end

  resources :sessions do
    collection do
      get :login
      post :login
      delete :logout
    end
  end

end
```

こう言う感じで考えたのですが、これだと名前付きパスが使用できないみたいなのでテストでエラーがでてしまうみたいでした.......

Reply:
その他のファイルも訂正しました！